src/userdb/userdbctl.c

```diff
@@ -8,15 +8,14 @@
 #include "fd-util.h"
 #include "format-table.h"
 #include "format-util.h"
-#include "group-record-show.h"
 #include "main-func.h"
 #include "pager.h"
 #include "parse-util.h"
 #include "pretty-print.h"
+#include "record-show.h"
```

Comment:
that's an awfully generic name, we have log records after all too, and the filename might suggest this has something to do with that? Maybe stick to "user-record-show.h"? (or user-group-record-show.h, if you must)

Reply:
`nss-record-show.[ch]` ?